### PR TITLE
add example plots for radar and radiometer data from unified dataset

### PR DESCRIPTION
**Pull request by JuleRadtke**
first example quicklooks from Heikes unified dataset

### CODE REVIEW

**Comment by tmieslinger** [comment]
Hi @JuleRadtke, thanks for adding the plots! 
it runs through smoothly :) I have two small suggestions to the second plot:

1. `for` loops in python are really nice as they can work based on collections, not necessarily numbers. So here, you want to loop over and plot the time series of selected frequencies. To keep the code simple you can best do the selection first and then loop over the `dataset` and `frequency`, such as
```python3
# 2nd plot: Radiometer TB
## select low frequency channels along the 22 GHz water vapor line
low_freq = ds_radiometer_selection.frequency < 32
ds_radiometer_low_freq = ds_radiometer_selection.isel(frequency=low_freq)

for frequency, data_radiometer in ds_radiometer_low_freq.groupby("frequency"):
    data_radiometer.tb.plot(ax=ax2, x='time', label=f'{frequency:.2f} GHz')
```
    You can shorten the label statement by using the format specifications instead of `numpy.round`.

2. as the lines correspond to increasing frequency, you could use a sequential colormap, for example:
```python3
## set line colors for 2nd plot
colors2 = plt.get_cmap("magma")(np.linspace(0, 1, low_freq.values.sum()))
ax2.set_prop_cycle(color=colors2)
```
    (you would have to put that before the loop though)

**Comment by tmieslinger** [comment]
@d70-t just added a pull request on the MyST conversion, which makes live easier, however, for you in the current pull request slightly more complicated. If you `fetch` and `merge` the current master you will get a merge conflict. The following will solve the conflict by converting your notebook to markdown and deleting the notebook:
```
jupytext --to=myst how_to_eurec4a/unified.ipynb
rm how_to_eurec4a/unified.ipynb
```

**Comment by JuleRadtke** [comment]
Hi @tmieslinger! Thanks for your comments!! The suggestions are great, I incorporated them. Additionally, I included a small plotting example from bahamas (plotting the flight altitude on top of the radar plot ). 
Lets see if I managed to avoid a merge conflict...

**Comment by d70-t** [comment]
Hey @JuleRadtke thanks for updating the PR :smile:! However, there's still a conflict. In order to resolve it, you should fetch the current `master` from @tmieslinger s "upstream" repository and merge it into your branch. This will display exactly the same conflict, but you can then resolve it locally (by accepting your updated version of `unified.md` and discarding what's in that file on the upstream repository).

Here's a screenshot from the [Network pane](https://github.com/tmieslinger/how_to_eurec4a/network) of the current situation:
image
You can see your branch in orange and there's also the long blue one which has been in the same situation. The violet branch was merged into `master` which created a conflict for the blue one. Then the `master` has been merged back into the blue one (the little black arrow) while solving the conflict and subsequently the blue one could be merged into `master` thus finishing the branch. This is exactly what's needed for the orange one now.

**Comment by JuleRadtke** [comment]
Hi @d70-t thanks, the network pane is great, it worked!

File: how_to_eurec4a/unified.md [hunk 1]
```diff
@@ -48,3 +49,60 @@ All HALO flights were split up into flight phases or segments to allow for a pre
...
+
+ax1.set_xlabel('')
+for ax in [ax1, ax2]:
+    ax.spines['right'].set_visible(False)
+    ax.spines['top'].set_visible(False)
+```
+
+```{code-cell} ipython3
+
+```
```
Comment (d70-t) [inline how_to_eurec4a/unified.md:108]: This last cell seems to be redundant.
```suggestion
```

Comment (JuleRadtke) [reply how_to_eurec4a/unified.md]: True!

**Comment by d70-t** [comment]
:+1: :smile:

**Pull request merged**